playground/worker/__tests__/relative-base/relative-base-worker.spec.ts

```diff
@@ -68,8 +68,9 @@ describe.runIf(isBuild)('build', () => {
     expect(workerContent).not.toMatch(`import`)
     expect(workerContent).not.toMatch(`export`)
     // chunk
-    expect(content).toMatch(`new Worker("../worker-entries/`)
-    expect(content).toMatch(`new SharedWorker("../worker-entries/`)
+    console.log(content)
```

Comment:
Is this intentional?

```suggestion
```

Reply:
Definitely not, thanks for the catch @tony19